roxctl/central/initbundles/fetch_ca.go

```diff
@@ -37,13 +37,9 @@ func fetchCAConfig(cliEnvironment environment.Environment, outputFile string,
 	if err != nil {
 		return errors.Wrap(err, "opening output file for writing CA config")
 	}
-	defer func() {
-		if bundleOutput != nil {
-			_ = bundleOutput.Close()
-			utils.Should(os.Remove(outputFile))
-		}
-	}()
 	if err := writeCA(ctx, svc, bundleOutput); err != nil {
+		_ = bundleOutput.Close()
+		utils.Should(os.Remove(outputFile))
```

Comment:
Really `Should`? Can't we just ignore the error here?

Reply:
previously we have should, in release it will be ignored